bot/config.py.zephyr.sample

```diff
@@ -87,15 +87,21 @@ z['iut_config'] = {
     "prj.conf": {},  # Default config file name
 
     "privacy.conf": {
-        "overlay": {'CONFIG_BT_PRIVACY': 'y'},
+        "overlay": {'CONFIG_BT_PRIVACY': 'y',
+                    'CONFIG_BT_SMP_ENFORCE_MITM': 'y'},
         "test_cases": ['GAP/PRIV/CONN/BV-10-C', 'GAP/DISC/LIMM/BV-03-C',
                        'GAP/DISC/GENM/BV-03-C', 'GAP/CONN/NCON/BV-02-C',
                        'GAP/CONN/NCON/BV-03-C', 'GAP/CONN/UCON/BV-06-C',
                        'SM/SLA/KDU/BV-02-C', 'SM/MAS/KDU/BV-05-C',
                        'GAP/CONN/ACEP/BV-03-C', 'GAP/CONN/DCEP/BV-05-C',
                        'GAP/CONN/GCEP/BV-05-C', 'GAP/BROB/BCST/BV-03-C',
                        'GAP/PRIV/CONN/BI-01-C', 'GAP/BROB/OBSV/BV-06-C',
-                       'GAP/PRIV/CONN/BV-11-C']},
+                       'GAP/PRIV/CONN/BV-11-C',
+                       'SM/MAS/KDU/BV-10-C', 'SM/MAS/KDU/BV-11-C',
+                       'SM/MAS/SCJW/BI-01-C', 'SM/SLA/KDU/BV-08-C',
+                       'SM/SLA/KDU/BV-09-C', 'SM/SLA/SCJW/BI-02-C',
+                       'SM/SLA/SCPK/BI-03-C', 'SM/SLA/SCPK/BI-04-C',
+                       'SM/SLA/SCPK/BV-02-C', 'SM/SLA/SCPK/BV-03-C',]},
```

Comment:
If I may, since we are now "touching" this file,
I would like to note that to the best of my understanding, the overlays should be: 
```python
    "enforce_mitm.conf": {
        "overlay": {'CONFIG_BT_SMP_ENFORCE_MITM': 'y'},
        "test_cases": ['SM/SLA/PKE/BV-05-C',
                       'SM/SLA/SCPK/BI-04-C',
                       'GAP/SEC/AUT/BV-12-C',
                       'GAP/SEC/AUT/BV-13-C'    
                       ]
    },
    "privacy.conf": {
        "overlay": {'CONFIG_BT_PRIVACY': 'y'},
        "test_cases": ['GAP/PRIV/CONN/BV-10-C',
                       'GAP/PRIV/CONN/BV-11-C',
                       'GAP/CONN/ACEP/BV-03-C', # As of PTS v7.6.2, not supported
                       'GAP/CONN/DCEP/BV-05-C', # As of PTS v7.6.2, not supported
                       'GAP/CONN/GCEP/BV-05-C', # As of PTS v7.6.2, not supported
                       'GAP/CONN/NCON/BV-02-C',
                       'GAP/CONN/UCON/BV-06-C',
                       'GAP/BROB/BCST/BV-03-C',
                       'GAP/BROB/OBSV/BV-06-C',
                       'SM/MAS/KDU/BV-05-C',
                       'SM/MAS/KDU/BV-10-C',
                       'SM/MAS/KDU/BV-11-C',
                       'SM/SLA/KDU/BV-02-C',
                       'SM/SLA/KDU/BV-08-C',
                       ]
    },
```

Notice that I have removed:
- GAP/PRIV/CONN/BI-01-C: It seems that this tc has been withdrawn

These tests are passing with the default prj.conf:
- SM/SLA/KDU/BV-09-C
- SM/MAS/SCJW/BI-01-C
- SM/SLA/SCJW/BI-02-C
- SM/SLA/SCPK/BV-02-C
- SM/SLA/SCPK/BV-03-C
- SM/SLA/SCPK/BI-03-C
- GAP/DISC/LIMM/BV-03-C
- GAP/DISC/GENM/BV-03-C
- GAP/CONN/NCON/BV-03-C

Added in enforce_mitm_conf:
- GAP/SEC/AUT/BV-12-C
- GAP/SEC/AUT/BV-13-C
- SM/SLA/PKE/BV-05-C

@aydu We have introduced a regression for the following tests,
in our ncs 1.4 qualification branch:
- SM/SLA/KDU/BV-02-C
- SM/SLA/KDU/BV-08-C

The Upper tester enables privacy but does not properly set a device name in the advertising data.
Thus, the lower is unable to connect.
The offending change is this:
https://github.com/AyturkDuzen/auto-pts/pull/4/files#diff-e87dd551d6013441cf7799fb88d06d0dc4cf40d8958e86a0cf2b6487aaed8ce8L121